### DISCUSSION TITLE
Thanks for the 0.25.1 Libretube update

### DISCUSSION THREAD

**Discussion by CDXmarcelo** [libre-tube/LibreTube#6403] [General]
Dunno if here is the right place for this kind of feedback, but thanks for the new update. Reading the FAQ, I got the impression that people put a lot of pressure on the project. And since I can't donate, thought I'd at least say that Libretube has been a lot of help to me since I started using it.

Now, let's download and test the new version.

**Comment by RichardOntaneda** [comment]
Una gran aplicación le configure en los 3 puntos en instancia y seleccione deshabilitar el próxy de Piped y seleccione las dos opciones y listo reinicie la app y todo correcto en la actualización v
![Screenshot_20240827_012310_LibreTube](https://github.com/user-attachments/assets/c3f5f569-e085-4d57-988f-503493be03cd)
0.25.1

**Comment by Bnyro** [comment]
Thanks for the kind words @CDXmarcelo, I appreciate it <3
